swap/swap.go

```diff
@@ -79,9 +82,24 @@ func NewParams() *Params {
 	}
 }
 
+// NewLogger returns a new logger
+func NewLogger(h interface{}) log.Logger {
+	swapLogger := log.New("swaplog", "*")
+	hd, ok := h.(log.Handler)
+
+	if !ok {
+		//default handler just in case the handler is not set
+		hd = log.LvlFilterHandler(log.Lvl(2), log.StreamHandler(colorable.NewColorableStderr(), log.TerminalFormat(true)))
```

Comment:
The default verbosity for Swarm is 3 (`cmd/global-store/main:58`). Is there a special reason you choose two instead of 3?

Reply:
I was not sure on the specific verbosity level.
I found the verbosityFlag is actually 3 as you stated,
I will correct this.

Maybe I could switch it to use the global verbosityFlag.

Reply:
While it's true that the verbosity default is 3, this is actually not where it is set (`global-store` is an independent binary used for debugging/testing, where a single shared chunk store can be used for different nodes).

It is set at ` internal/debug/flags.go:42`, set via `cmd/swarm/main.go:217`

Reply:
This probably would not need to be exported, as it is only called from within the `swap` package no?

Reply:
It's only for the swap package.